### PR TITLE
Pixel flappy bird

### PR DESCRIPTION
**Pull request by flipps**
This is my Flappy bird game logo. I really enjoyed the logo that Daibushi did! But my attempt was to bring that pixel art feeling back to the logo :). Hope you like it, and if you think it should need some more tweaking, just let me know! It was pretty fun to make it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +13,7 @@ Although it's still incomplete, you can play it here: [https://andrevarandas.git
 1. Clone this repository:
 `git clone https://github.com/AndreVarandas/the-flappy-bird`
 
-2. Change directory into `the-flappy-bird` and install dependencies:
+2. Change directory into `the-flappy-bird` and install dependencies:1
```
Comment (AndreVarandas) [inline README.md]: The 1  is a typo?

**Comment by AndreVarandas** [comment]
Thanks for your MR @flipps! I love it!
 Please see my comment, just a small change needed there 👍

File: README.md (hunk 1 above)
Comment (flipps) [reply README.md]: Don't know how it got there, but yeah! Going to correct this!

Comment (flipps) [reply README.md]: Done, made a new commit to correct the problem :)

**Pull request merged**